core/src/main/java/org/mule/config/builders/DefaultsConfigurationBuilder.java

```diff
@@ -106,6 +106,7 @@ protected void doConfigure(MuleContext muleContext) throws Exception
         registry.registerObject(MuleProperties.OBJECT_EXPRESSION_LANGUAGE, new MVELExpressionLanguageWrapper(muleContext));
         registry.registerObject(MuleProperties.OBJECT_CONNECTOR_MESSAGE_PROCESSOR_LOCATOR, new MuleConnectorOperationLocator());
         registry.registerObject(MuleProperties.OBJECT_TIME_SUPPLIER, new TimeSupplier());
+        registry.registerObject(MuleProperties.OBJECT_MULE_CONTEXT, muleContext);
```

Comment:
Why put muleContext into the registry? Or is it already there, and just missing from this config builder?